NGitLab/Models/SubgroupQuery.cs

```diff
@@ -0,0 +1,63 @@
+﻿namespace NGitLab.Models
+{
+    /// <summary>
+    /// Allows to use more advanced GitLab queries for getting subgroups (based on v4 GitLab API).
+    /// https://docs.gitlab.com/ee/api/groups.html#list-a-groups-subgroups
+    /// </summary>
+    public class SubgroupQuery
+    {
+        /// <summary>
+        /// Skip the group IDs passed
+        /// (optional)
+        /// </summary>
+        public int[] SkipGroups;
```

Comment:
Use properties everywhere (using fields is _legacy_)

```suggestion
        public int[] SkipGroups { get; set; }
```

Reply:
Done 😄